drivers/xen/gnttab.c

```diff
@@ -324,46 +331,150 @@ const char *gnttabop_error(int16_t status)
 	}
 }
 
-static int gnttab_init(void)
+static int map_grant_frames(unsigned int start_frame, unsigned int end_frame)
```

Comment:
This function is always called with `(x, x+1)` parameter. Can we rename it to `map_grant_frame(unsigned int start_frame)`? And remove `for` loop inside.